app/code/Magento/ProductVideo/view/adminhtml/web/js/new-video-dialog.js

```diff
@@ -1103,6 +1104,7 @@ define([
                 this._previewImage = null;
             }
             this._tempPreviewImageData = null;
+            this._videoUrlWidget.trigger('cancelled_video_information');
```

Comment:
Hello @gixid192 ,
thank you for your contribution.

I have two questions for you:

a. Can you write a test for this PR?
b. I have found a new bug, I tell you a step to reproduce the bug:

b.1. Navigate to Add New Product page in backend
b.2. Open Image And Videos section
b.3. Click on Add Video button
b.4. Enter Vimeo or YouTube video URL in Url field and wait until the Save button is enabled
b.5. Click on Save button
b.6. Click on Add Video Button
b.7. paste URL in Url input, click Title and quickly click on X to close
b.8. Press on Add Video button
b.9. See an Preview Image (bottom left corner) from the previous video

![image](https://user-images.githubusercontent.com/13854502/105750927-9dd07a80-5f45-11eb-8ae2-54c9ec405e6c.png)

Reply:
Hi @danielrussob,
1. I'm not sure how to write test for this case. Could you suggest some ideas?
2. Did you try this new case with the code in the PR or the original code? With the PR's code, it stop the render part right after you close X, so there will be no b.8/b.9
I tried it myself, couldn't reproduce the issue.

Reply:
Hi,
for point 2, i have tested it on your PR, if you want we can schedule a chat on slack of Magento Community Engineering
 to reproduce the issue (you can found me with name Daniel Russo)

for point 1, see comment of @sivaschenko

Reply:
Hi,
So I tried again and reproduced the issue.

There's another ajax in the update image-placeholder: https://github.com/magento/magento2/blob/cd857f950dce201b1a496843013a7f1e200bced1/app/code/Magento/ProductVideo/view/adminhtml/web/js/new-video-dialog.js#L393

I made the same approach and it worked! However, I don't understand the purpose of that `_loadRemotePreview` function.
The `_loadRemotePreview` function does 2 things:
- Send request to server to save the image
- Only show the preview_image if the request above success.

The thing is those 2 things are not related, it does not take data from the request to show for users (instead the preview_image is taken from video's api). Also, if the request fail, there is no error handle for it (and the preview_image still show)

Should we split this function into 2 small separated ones?

Reply:
@danielrussob: I updated new code to fix the issue you reported, just write new code, no splitting; kindly have a look when you have time.

If you have options on this `split into 2 functions`, please let me know.